lavalink/Events.py

```diff
@@ -30,13 +34,29 @@ def __init__(self, player, track, reason):
 
 class TrackStartEvent:
     """ This event will be dispatched when the player starts to play a track. """
+
     def __init__(self, player, track):
         self.player = player
         self.track = track
 
 
 class StatsUpdateEvent:
     """ This event will be dispatched when the websocket receives a statistics update. """
+
     def __init__(self, node):
         self.stats = node.stats
         self.node = node
+
+
+class NodeReadyEvent:
+    """ This event will be dispatched when the node is ready for use. """
+
+    def __init__(self, node):
+        self.node = node
+
+
+class NodeDisabledEvent:
+    """ This event will be dispatched when the node is removed from service due to an error. """
+
+    def __init__(self, node):

```

Comment:
Would it be possible to include the reason for disabling?

Reply:
Possible but (as of now) there's only one reason: ws disconnect